### ISSUE TITLE
Helm causes relative line numbers to become "1" with linum-relative

### ISSUE THREAD

**Issue by dudelson** [syl20bnr/spacemacs#6493]
#### Description
When relative line numbers are enabled using the linum-relative package, opening any sort of helm buffer (e.g. Helm find files, Helm mini, Helm M-x, etc.), causes all line numbers in all visible buffers to become the numeral "1".

#### Reproduction guide
- Start Emacs
- Enable relative line numbers if you haven't already (`dotspacemacs-line-numbers 'relative`)
- Open any non-empty buffer
- From the buffer, open any helm buffer (`SPC f f`, `SPC b b`, `SPC :`, etc.)

*Observed behaviour:*
All line numbers in all visible buffers become the numeral "1". Closing the helm buffer restores the relative line numbers in all buffers.

*Expected behaviour:*
Relative line numbers should be preserved in all buffers while any helm buffer is open.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.21
- Spacemacs branch: master (rev. 0283f64)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-return-key-behavior nil auto-completion-tab-key-behavior 'cycle :disabled-for org)
 c-c   emacs-lisp git gtags lua org rust semantic shell-scripts syntax-checking unimpaired)
```

I have solved this issue by using nlinum and nlinum-relative in place of linum and linum-relative (thanks @a13ph!), however I was asked to document the issue and its solution. It looks like nlinum is already in the develop branch, so this fix is only temporary, but for now on stable spacemacs I added the following to my config:

```lisp
    ;; in dotspacemacs/layers
   dotspacemacs-additional-packages
   '(
     nlinum
     nlinum-relative
    )
    ...
    ;; in dotspacemacs/init
    dotspacemacs-line-numbers nil
    ...
    ;; in dotspacemacs/user-config
    (require 'nlinum-relative)
    (nlinum-relative-setup-evil)                    ;; setup for evil
    (add-hook 'prog-mode-hook 'nlinum-relative-mode)
    (add-hook 'text-mode-hook 'nlinum-relative-mode)
    (setq nlinum-relative-redisplay-delay 0)      ;; delay
    (setq nlinum-relative-current-symbol "")      ;; or "" for display current line number
    (setq nlinum-relative-offset 0)
    (setq nlinum-format "%d ")
```

In the future I think I should be able to simply add the nlinum layer.

**Comment by istinspring** [comment]
ah yes, can confirm this problem on master.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
